pkg/controller/hyperconverged/factory.go

```diff
@@ -0,0 +1,149 @@
+package hyperconverged
+
+import (
+	sspv1 "github.com/MarSik/kubevirt-ssp-operator/pkg/apis/kubevirt/v1"
+	networkaddons "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+	networkaddonsnames "github.com/kubevirt/cluster-network-addons-operator/pkg/names"
+	hcov1alpha1 "github.com/kubevirt/hyperconverged-cluster-operator/pkg/apis/hco/v1alpha1"
+	kwebuis "github.com/kubevirt/web-ui-operator/pkg/apis/kubevirt/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	cdi "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
+	kubevirt "kubevirt.io/kubevirt/pkg/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+// The set of resources managed by the HCO
+func (r *ReconcileHyperConverged) getAllResources(cr *hcov1alpha1.HyperConverged, request reconcile.Request) []runtime.Object {
+	return []runtime.Object{
+		newKubeVirtConfigForCR(cr, request.Namespace),
+		newKubeVirtForCR(cr, request.Namespace),
+		newCDIForCR(cr, ""),
+		newNetworkAddonsForCR(cr, ""),
+		newKubeVirtCommonTemplateBundleForCR(cr, "openshift"),
+		newKubeVirtNodeLabellerBundleForCR(cr, request.Namespace),
+		newKubeVirtTemplateValidatorForCR(cr, request.Namespace),
+		newKWebUIForCR(cr, ""),
+	}
+}
+
+func newKubeVirtConfigForCR(cr *hcov1alpha1.HyperConverged, namespace string) *corev1.ConfigMap {
+	labels := map[string]string{
+		"app": cr.Name,
+	}
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kubevirt-config",
+			Labels:    labels,
+			Namespace: namespace,
+		},
+		Data: map[string]string{
+			"feature-gates": "DataVolumes,SRIOV,LiveMigration,CPUManager,CPUNodeDiscovery",
+		},
+	}
+}
+
+// newKubeVirtForCR returns a KubeVirt CR
+func newKubeVirtForCR(cr *hcov1alpha1.HyperConverged, namespace string) *kubevirt.KubeVirt {
+	labels := map[string]string{
+		"app": cr.Name,
+	}
+	return &kubevirt.KubeVirt{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kubevirt-" + cr.Name,
+			Labels:    labels,
+			Namespace: namespace,
+		},
+	}
+}
+
+// newCDIForCr returns a CDI CR
+func newCDIForCR(cr *hcov1alpha1.HyperConverged, namespace string) *cdi.CDI {
+	labels := map[string]string{
+		"app": cr.Name,
+	}
+	return &cdi.CDI{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "cdi-" + cr.Name,
+			Labels:    labels,
+			Namespace: namespace,
```

Comment:
Shouldn't this always be `""`?  Line 154 calls `cdi := newCDIForCR(instance, request.Namespace)`, which may not always be correct.  Maybe we should create some `const`s with NS values and spread them around?

Reply:
Good catch. Yes, constants are a good idea. Added.